src/libraries/System.Speech/src/System.Speech.Internal.SapiInterop/SapiProxy.cs

```diff
@@ -149,7 +149,6 @@ private void Dispose(bool disposing)
                     });
                     ((IDisposable)_process).Dispose();
                     ((IDisposable)_done).Dispose();
-                    _mta.Abort();
```

Comment:
Here's another PNSE to avoid.  I just removed this call and added a catch for the ObjectDisposedException that'll happen to let the thread die gracefully.

Reply:
Ugh, how could I have found these - did I not run the right analyzer?